RecoTracker/SpecialSeedGenerators/src/CtfSpecialSeedGenerator.cc

```diff
@@ -17,6 +17,13 @@ using namespace ctfseeding;
 
 CtfSpecialSeedGenerator::CtfSpecialSeedGenerator(const edm::ParameterSet& conf)
     : conf_(conf),
+      mfToken(esConsumes<edm::Transition::BeginRun>()),
```

Comment:
@makortel 
what is the lifetime of the data declared to be consumed during a transition like `edm::Transition::BeginRun` here?
Is the MF pointer cached during beginRun going to be still available during the event processing?

Reply:
The lifetime of an ESProduct (that is accessed via something else than `edm::ESTransientHandle`) is guaranteed until the end of the IOV.

The impact of `edm::Transition::BeginRun` is to framework to guarantee that the ESProduct has been produced before the beginRun of this module is called.

Reply:
I assume there's not follow-up needed here.

Reply:
right